src/LiveDevelopment/MultiBrowserImpl/transports/node/NodeSocketTransportDomain.js

```diff
@@ -58,7 +58,7 @@
     var _clients = {};
     
     // This must match the port declared in NodeSocketTransport.js.
-    // TODO: randomize this?
+    // Default socket port number
     var SOCKET_PORT = 8123;
```

Comment:
Are you sure? This is part of NodeDomain, can I read prefs there? Anyway: starting server sends port number, this here will be overwritten.